src/components/drawer/drawer.component.js

```diff
@@ -2,39 +2,46 @@ import React, { useState, useCallback, useRef, useEffect } from "react";
 import PropTypes from "prop-types";
 import classNames from "classnames";
 import invariant from "invariant";
-import createGuid from "../../../utils/helpers/guid/guid";
-import Icon from "../../icon";
 
+import createGuid from "../../utils/helpers/guid/guid";
+import Icon from "../icon";
 import {
+  StyledSidebarHeader,
   StyledDrawerWrapper,
   StyledDrawerContent,
   StyledButton,
   StyledDrawerChildren,
   StyledDrawerSidebar,
   StyledSidebarTitle,
 } from "./drawer.style";
+import StickyFooter from "../../__internal__/sticky-footer";
 
 const SidebarContext = React.createContext();
 
 const Drawer = ({
-  defaultExpanded,
+  defaultExpanded = true,
   expanded,
   onChange,
   children,
-  expandedWidth,
+  expandedWidth = "40%",
   sidebar,
-  animationDuration,
+  animationDuration = "400ms",
```

Comment:
I wonder why you didn't use the default props. Is this the approach I should take as well?

Reply:
Yeah this is a cleaner way of doing it 👍🏻 I think it's better having the defaults at the top of the component like this as well, rather than having to scroll down below it to see what the defaults are

Reply:
and using `defaultProps` on functional components is going to be deprecated in React at some point as well!

Reply:
Great, thanks a lot!